extensions/-Microboy-/geminiExtension.js

```diff
@@ -0,0 +1,66 @@
+// Name: Gemini Extension
+// ID: geminiExtension
+// Description: Allows using Google Gemini within Scratch projects. In fact, this extension is generated by Google Gemini itself!
```

Comment:
> In fact, this extension is generated by Google Gemini itself!
We **seriously** prohibit extensions that are generated by AI and not created by someone who has any prior JavaScript experience. 

Language models do not understand how to make Scratch extensions, let alone TurboWarp extensions which have a whole other set of needs.

What you've done here is just admitted that you haven't read any of the TurboWarp extension documentation, nor the contributing guidelines. Quite a self report, don't you think?

For that very reason I'm not even going to review this.